app/src/main/java/eu/kanade/tachiyomi/data/track/anilist/dto/ALManga.kt

```diff
@@ -60,6 +61,7 @@ data class ALUserManga(
         last_chapter_read = chaptersRead.toDouble()
         library_id = libraryId
         total_chapters = manga.totalChapters
+        private = private
```

Comment:
```suggestion
        private = this@ALUserManga.private
```
Found it. Otherwise, remoteManga will have false here, and we update our local one with the data from remote.

Reply:
ahhhh damn that was sneaky, nice job